addons/web/views/speedscope_config_wizard.xml

```diff
@@ -0,0 +1,77 @@
+                                <label class="form-check-label" for="combined_profile">Combined profile</label>
+                            </li>
+                            <li class="form-check">
+                                <input type="checkbox" class="form-check-input" id="combined_no_context_profile" name="combined_no_context_profile" value="True" checked="checked"/>
+                                <label class="form-check-label" for="combined_no_context_profile">Combined no context profile</label>
+                            </li>
+                            <li class="form-check">
+                                <input type="checkbox" class="form-check-input" id="sql_no_gap_profile" name="sql_no_gap_profile" value="True" checked="checked"/>
+                                <label class="form-check-label" for="sql_no_gap_profile">SQL no gap profile</label>
+                            </li>
+                            <li class="form-check">
+                                <input type="checkbox" class="form-check-input" id="sql_density_profile" name="sql_density_profile" value="True" checked="checked"/>
+                                <label class="form-check-label" for="sql_density_profile">SQL Density profile</label>
+                            </li>
+                            <li class="form-check">
+                                <input type="checkbox" class="form-check-input" id="frames_profile" name="frames_profile" value="True" checked="checked"/>
+                                <label class="form-check-label" for="frames_profile">Frames profile</label>
+                            </li>
+                        </ul>
+                    </div>
+
+                    <input type="hidden" name="profile_id" t-att-value="profile.id" />
+                    
+                    <div class="mt-3">
+                        <button type="button" class="btn btn-primary" onclick="submitForm('/web/content/ir.profile', true)">Download</button>
+                        <button type="button" class="btn btn-secondary" onclick="submitForm('/web/speedscope', false)">Open</button>
+                    </div>
+                </form>
+            </div>
+        </div>
+
+        <script>
+            function submitForm(actionUrl,download) {
+                const form = document.getElementById('dynamicForm');
+                const formData = new FormData(form);
+                const profileID = formData.get('profile_id')
+                formData.delete('profile_id')
+                const queryString = new URLSearchParams(formData).toString();
+                const fullUrl = download
+                    ? `${actionUrl}/${profileID}/speedscope?${queryString}&amp;download=${download}&amp;filename=profile_${profileID}.json`
+                    : `${actionUrl}/${profileID}?${queryString}`;
+                window.location.href = fullUrl;
+            }
+        </script>
```

Comment:
Not necessary with the following changes ([local diff](https://gist.github.com/Williambraecky/f582ec1aa541a6c0718c53ccbc8e6f01)):
- Add a target on the `form` tag (`/web/speedscope/{{profile.id}}/`)
- Change the two buttons to input (name="action" value="download|open")
```xml
<input type="submit" class="btn btn-primary" name="action" value="download"/>
```
- Remove hidden profile_id input (it's not necessary)
- Support the download action in the speedscope controller by redirecting to the /web/content route